frontend/src/views/MigrationHistoryDetail.vue

```diff
@@ -180,7 +241,10 @@ import {
 } from "../types";
 
 interface LocalState {
-  showDiff: boolean;
+  // showBaselineChange switches show diff between *lastRecordedSchema* and schema.
```

Comment:
Please describe the show/hide/diff logic for different migration types since it's not that straightforward.

By expressing the thought, other folks can learn and may also spot the potential improvement.

Reply:
I refactor code diff as a component so that the logic of 1. should this part be shown and 2. switch on/off comparison can be separated